### ISSUE TITLE
Implement new proposers web pages

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/pdsen-operations#28]
**Is your feature request related to a problem? Please describe.**
Implement new Data Proposers web pages.

Will replace https://pds-gamma.jpl.nasa.gov/home/proposers/ 

Here is the info for the web pages (NOTE: this is still TENTATIVE. will comment on ticket when ready to go for implementation):

Main page: https://docs.google.com/document/d/1ckzrcD5-mTEjjJyemeMLr8NRQaQj8DDQLjhHPdyhWqM/edit#
Links to a couple other pages:
* [Guidelines for Archiving in the PDS](https://docs.google.com/document/d/137G7L-6nAEc5krBxX3H5qSx3x5w0b_iQkqms_eJZ99g/edit?usp=sharing)
   * Links to: [Why Start Archiving Process Early?](https://docs.google.com/document/d/1PZEsrmO_xbFqqALN8lAzY2__aVQxc8xJ1AjQEwQe9NQ/edit?usp=sharing)
* [PDS Letter of Support Request](https://docs.google.com/document/d/1sI4lJEjporQsoWDp4eOFbXKC0s6tXZjeOpeVqihASFI/edit?usp=sharing) (should link to downloadable word doc. we will not be implementing a web form)

**Features to keep in mind:**
* Web page side bar - should mimic the side bars on the actual google docs for ease of navigation

**Plan:** to have this implemented by week of 12/2 sprint end, review on 12/9, and rollout with Build 11.0 web updates

**Comment by myche** [comment]
Is the intent for this new Data Proposer's page to be one-stop-shopping or will it continue to link to the specific Node's pages?

**Comment by jordanpadams** [comment]
@myche hopefully one stop shopping. It is up to each node, but we would encourage nodes to direct people here unless they have explicit information that requires DN curation. Even then, we hope to eventually be able to house all that info at PDS.nasa.gov

**Comment by c-suh** [comment]
@jordanpadams content and beginnings of basic style and functionality have been posted to https://pds-gamma.jpl.nasa.gov/home/proposers/. Apologies for the delay!

**Comment by jordanpadams** [comment]
thanks @c-suh looks great! a few comments:

1. Can we remove that separate "PDS Nodes" sidebar along the left?
2. Can we also remove all the FAQs from the Table of Contents along the left panel, and instead put a list of the FAQs with links at the beginning of the FAQs section?
3. For the second FAQ question, can we bold the summary: for each bullet (e.g. **Long-term preservation:**)?
4. This may seem silly, but can we also increase the line spacing in the paragraph text? seems kind of hard to read
5. Looks like the menu bars are kind of messed up.
6. also looks like there have been a few changes to the doc since you completed the development. They look pretty trivial, but if you go to back to the doc, and click the link at the top "Last edit was made X hours ago by Nancy Chanover", that will show you the changes she completed this afternoon.

@edwardbarraza any thoughts? obviously we are trying to stay within the current web-design framework, but even a little guidance would be helpful

**Comment by c-suh** [comment]
> 1. Can we remove that separate "PDS Nodes" sidebar along the left?

Done

> 2. Can we also remove all the FAQs from the Table of Contents along the left panel, and instead put a list of the FAQs with links at the beginning of the FAQs section?

Done; however, is it redundant with the page's list of contents (which includes the list of FAQs) just above it in the Overview section?

> 3. For the second FAQ question, can we bold the summary: for each bullet (e.g. Long-term preservation:)?

Done

> 4. This may seem silly, but can we also increase the line spacing in the paragraph text? seems kind of hard to read

Done (for both paragraphs and lists); is more or less preferred?

> 5. Looks like the menu bars are kind of messed up.

Fixed

> 6. Remove "Overview" from TOC

Done; but keep in the Overview's list of contents?

> 7. also looks like there have been a few changes to the doc since you completed the development. They look pretty trivial, but if you go to back to the doc, and click the link at the top "Last edit was made X hours ago by Nancy Chanover", that will show you the changes she completed this afternoon.

I know what link you mean but am not seeing it at the moment. I will try again tomorrow.

> 8. For the button the "Form for a Letter of Support", let's just make this say "Request Form". also, let's download the latest version of the this doc and actually make this a downloadable word doc to be filled out

Done (button text) and Will do (downloadable .docx).

> also I am assuming the Guidelines for Archiving page and the links to it will be updated once that page is ready?

Didn't see the link before but it's there now! Is this to be another page under the Data Proposers tab?

---

To recap: on my todo list
- [ ]  update with Nancy's latest edits and any other that I may have missed
- [ ]  create and link to a downloadable .docx Request for a Letter of Support
- [ ]  create new page, Guidelines for Archiving ?

**Comment by edwardbarraza** [comment]
If we are limiting the left-hand sidebar for the FAQ topics, then let's organize sub=topics within each designated section. The list of anchors to each topic and subtopic in the Overview section is redundant. Getting rid of it would clean up the page and we get rid of the redundancy. 

Normally, I'd recommend to put each topic and sub-topic in a TOC, but I think some of the FAQ's are a bit verbose and I suspect it won't work well within the bounds of the side-bar. 

I'd say just get rid of the list of topics and subtopics in the Overview section.

**Comment by jordanpadams** [comment]
@c-suh when you have a chance, can get back to this TODO list?

- [ ] update with Nancy's latest edits and any other that I may have missed
- [ ] create and link to a downloadable .docx Request for a Letter of Support
- [ ] create new page, Guidelines for Archiving - https://docs.google.com/document/d/137G7L-6nAEc5krBxX3H5qSx3x5w0b_iQkqms_eJZ99g/edit 
- [ ] Add FAQ TOC under the FAQ section - otherwise tracking down the FAQs are difficult to get through

**Comment by c-suh** [comment]
@jordanpadams please review the changes and let me know preference for the use of mini `<hr />`s within [Proposer's FAQs](https://pds-gamma.jpl.nasa.gov/home/proposers/#proposers-faqs) (inserted) and [Archiving's Generating Steps](https://pds-gamma.jpl.nasa.gov/home/proposers/archiving/#archiving-committing_to_generat)e (not inserted). I hope to also style the FAQ table of contents a little more obviously, but for now, I think the checklist above has been addressed.

**Comment by jordanpadams** [comment]
@c-suh looks great. can we make the FAQ TOC just a bulleted list instead of that enclosed table?

**Comment by jordanpadams** [comment]
@c-suh a few other mods from our meeting with the team today:

[ ] - FAQ TOC just a bulleted list instead of that enclosed table
[ ] - For the `Where in the PDS should my data go?` FAQ, please turn the node list into a bulleted list.
[ ] - For the `Where in the PDS should my data go?` FAQ, reword and reformat the `Don't Know` bullet to this:
     **Don't Know** - If you are unsure, for example if your data may be applicable to multiple nodes, submit the form below (link here) with your best guess, and the PDS will help guide you in the right direction.
[ ] - Replace `Overview` and `Detailed Archiving Checklist` with copy-paste from Archiving Milestones document (attachment TBD)
 [ ] - Replace request form with revised form from Nancy

**Comment by c-suh** [comment]
@jordanpadams the checked items are ready for review at
- https://pds-gamma.jpl.nasa.gov/home/proposers/ and 
- https://pds-gamma.jpl.nasa.gov/home/proposers/archiving/

Regarding the second to last item on the list, I was confused for an embarrassingly long time before realizing the content of the Archiving Milestones document is a closer match to `Committing to Generate a PDS Archive` than it is to `Detailed Archiving Checklist`. I went ahead as such, but let me know if I misunderstood or missed anything. Thank you.

**Comment by c-suh** [comment]
@jordanpadams I also took a stab at having the sidebar's Table of Contents indicate the section in view, since some of the text can get long. Let me know if this is agreeable (and should be applied to other pages with the sidebar's Table of Contents) or if I should remove it.

**Comment by jordanpadams** [comment]
@c-suh i actually intended to remove the checklist, but it may still work there. will send out to the group and see what they say

**Comment by jordanpadams** [comment]
@c-suh this should be the "final" updates before we can go live:

- [ ] Under Develop a Data Management Plan:

The links in the first bullet don't seem to work. They should link to
https://science.nasa.gov/researchers/sara/faqs/dmp-faq-roses
and 
https://science.nasa.gov/researchers/templates-planetary-science-division-appendix-c-roses-proposals

- [ ] In our FAQ section near the top, under "Is the PDS the right archive for me," we don't want to limit ourselves to ground-based observations that are acquired in support of NASA missions.​ Rather, we want to be more inclusive (we'd take ground-based observations that are NOT directly in support of a mission, for example).  Suggested rewording:

> ... the PDS archives data from NASA’s interplanetary missions as well as higher order data products derived from those primary mission data, and ground-based observations, laboratory and analog field measurements acquired in support of NASA's planetary science goals and objectives.

- [ ] In that same section, after the sentence "If you determine... more appropriate for your data." please add the following sentence:

> Proposers are also encouraged to review the ROSES Appendix C.1 section on Data Management Plans and Archiving for guidance concerning software, physical specimens, and other forms of data not archived by the PDS.

- [ ] On the Guidelines for Archiving page in the Overview section near the top in the Important Reminder​ text: change the first sentence to "While all projects vary in time and complexity, it is vital to start the PDS data archiving process early and to distribute the effort across the length of the project, even if you expect to acquire or generate data through the entire duration of the award period."

**Comment by c-suh** [comment]
@jordanpadams, the pages are ready for review and to be made operational.

**Comment by jordanpadams** [comment]
@c-suh we are a GO for deploying to operations.

**Comment by c-suh** [comment]
Task of deploying changes to operations is being tracked separately. This task of implementing said changes has been completed.
